src/trix/elements/trix_editor_element.coffee

```diff
@@ -24,6 +24,13 @@ Trix.registerElement "trix-editor", do ->
     return if element.hasAttribute("role")
     element.setAttribute("role", "textbox")
 
+  ensureAriaLabel = (element) ->
+    return if (element.hasAttribute("aria-label") or element.hasAttribute("aria-labelledby"))
+    return if (element.labels.some (label) -> label.contains(element))
```

Comment:
@javan I've added this guard clause to skip attempting to generate an `aria-label` value if the `<label>` includes the `<trix-editor>`, since the toolbar's contents would be a part of the `textContent`.